File: Latihan3-cpp/Disk.cpp
#include <string>

using namespace std;

class Disk{

    private:

    string type;
    int capacity;
    int price;

    public:

    // Constructor
    Disk(){
    }

    Disk(string type, int capacity, int price){
        this->type = type;
        this->capacity = capacity;
        this->price = price;
    }

    // Get-set Type
    void setType(string type){
        this->type = type;
    }

    string getType(){
        return this->type;
    }

    // Get-set Capacity
    void setCapacity(int capacity){
        this->capacity = capacity;
    }

    int getCapacity(){
        return this->capacity;
    }

    // Get-set Price
    void setPrice(int price){
        this->price = price;
    }

    int getPrice(){
        return this->price;
    }

    // Deconstructor
    ~Disk(){
    }

};

File: Latihan3-py/Disk.py
class Disk:
    # Constructor
    def __init__(self):
        self.__type = ""
        self.__capacity = 0
        self.__price = 0
    
    # Get-set Type
    def setType(self, type):
        self.__type = type

    def getType(self):
        return self.__type

    # Get-set Capacity
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity

    # Get-set Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price